src/ui/ui_menu.c

```diff
@@ -914,6 +914,49 @@ void  Menus_Activate(menuDef_t *menu)
 		DC->startBackgroundTrack(menu->soundName, menu->soundName, 0);
 	}
 
+	// register accelerator onKey action
+	for (i = 0; i < menu->itemCount; i++)
+	{
+		itemDef_t *item = menu->items[i];
+		const char *s   = item->text;
+
+		if (s == NULL) {
+			continue;
+		}
+
+		int       len                  = Q_UTF8_Strlen(s);
+		int       count                = 0;
+		short int accelerator          = 0;
+		qboolean  accelerator_upcoming = qfalse;
```

Comment:
This doesn't respect C89 standard. We try to stick as much as possible to this declaration style for concistency with code. Variables should be declared at top of the scope then `len` assigned after `s` NULL check.

Reply:
:metal: Done